melos.yaml

```diff
@@ -68,7 +68,7 @@ scripts:
 
   analyze:
     run: melos exec -c 1 --fail-fast -- \
-      flutter analyze
+      flutter analyze --no-fatal-infos
```

Comment:
@Jordan-Nelson The new analyzer bundled with Flutter 2.5 has changed a bit causing analysis to fail for missing public docs. I think `no-fatal-infos` is a good balance here - the analyzer will continue to show squiggly lines in the editor to remind you as you work, but not fail here if there are good reasons to leave them out. Same with the other "info" members.

And we can adjust rules to `warning` level if needed so that it does fail here.